src/main/java/vivatech/compat/rei/widget/CottonSlotWidget.java

```diff
@@ -15,19 +16,19 @@
     private final int x;
     private final int y;
 
-    private CottonSlotWidget(int x, int y, List<ItemStack> itemList, boolean showToolTips, boolean clickToMoreRecipes, WItemSlot renderSlot) {
-        super(x, y, itemList, false, showToolTips, clickToMoreRecipes);
+    private CottonSlotWidget(int x, int y, Renderer renderer, boolean showToolTips, boolean clickToMoreRecipes, WItemSlot renderSlot) {
+        super(x, y, renderer, false, showToolTips, clickToMoreRecipes);
```

Comment:
Not strictly related to this PR, but I'm not sure if `CottonSlotWidget` is needed anymore. I originally made it for painting the slots using the LibGui/Cotton style slot painter.

@Boundarybreaker Does Vivatech use the vanilla style slot painters now?

Reply:
I am not sure, but if it is not required, I can remove this.

Reply:
I haven't yet touched the VT gui stuff, but it will be shifting to vanilla-style slot painters when guis are touched.